sandbox/client.go

```diff
@@ -2,54 +2,40 @@ package sandbox
 
 import (
 	"crypto/rsa"
-	"crypto/x509"
 	"encoding/json"
-	"encoding/pem"
 	"fmt"
 	"io/ioutil"
 	"net/http"
 	"os"
 
+	yoti "github.com/getyoti/yoti-go-sdk/v2"
 	yotirequest "github.com/getyoti/yoti-go-sdk/v2/requests"
 )
 
-// Client is responsible for setting up test data in the sandbox instance
+// Client is responsible for setting up test data in the sandbox instance. BaseURL is not required.
 type Client struct {
-	AppID   string
-	Key     *rsa.PrivateKey
+	// Client SDK ID. This can be found in the Yoti Hub after you have created and activated an application.
+	ClientSdkID string
+	// Private Key associated for your application, can be downloaded from the Yoti Hub.
+	Key *rsa.PrivateKey
+	// Base URL to use. This is not required, and a default will be set if not provided.
 	BaseURL string
+	// Mockable HTTP Client Interface
+	httpClient yoti.HttpClient
 }
 
-// LoadPEMFile loads the Sandbox Client Key from a PEM file path
-func (client *Client) LoadPEMFile(filename string) error {
-	file, err := os.Open(filename)
-	if err != nil {
-		return err
-	}
-
-	buffer, err := ioutil.ReadAll(file)
-	if err != nil {
-		return err
-	}
-
-	block, _ := pem.Decode(buffer)
-	if block == nil {
-		return fmt.Errorf("Could not decode PEM file")
-	}
-	key, err := x509.ParsePKCS1PrivateKey(block.Bytes)
-	if err != nil {
-		return err
+// SetupSharingProfile creates a user profile in the sandbox instance
+func (client *Client) SetupSharingProfile(tokenRequest TokenRequest) (token string, err error) {
+	if client.BaseURL == "" {
+		if os.Getenv("API_URL") != "" {
```

Comment:
Should this be prefixed, e.g. `YOTI_API_URL` or `YOTI_API_BASE_URL`?

Reply:
Yep 👍 have changed to `YOTI_API_URL` in 22b60d1, so that we're consistent across SDKs